docs/en/3-api.md

```diff
@@ -128,7 +128,8 @@ var templates = bemxjst.bemhtml.compile(function() {
         // Setting up BEM naming
         naming: {
             elem: '__',
-            mod: '_'
+            mod: '_',
+            modVal: '--'
```

Comment:
Bad example. More common version for non-ya opensource is `block__elem--mod_val`